src/main/scala/org/broadinstitute/dsde/agora/server/webservice/validation/AgoraValidation.scala

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.dsde.agora.server.webservice.validation
+
+import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+
+/**
+ * Created by dshiga on 5/18/15.
+ */
+
+object AgoraValidation {
+  def validateMetadata(entity: AgoraEntity): AgoraValidation = {
+    var validation = AgoraValidation()
+    validation = entity.namespace.exists(_.trim.nonEmpty) match {
+      case false => validation.addError("Namespace is required")
+      case true => validation
```

Comment:
since validation is var anyways it might be a bit cleaner to just use an if here and add the message.

if(!entity.namespace.exists(_.trim.nonEmpty)) validation.addError("Namespace is required)